src/functional_test_runner/cli.js

```diff
@@ -75,6 +75,7 @@ const functionalTestRunner = createFunctionalTestRunner({
       bail: cmd.bail,
       grep: cmd.grep,
       invert: cmd.invert,
+      require: `ts-node/register --project tests/tsconfig.json -r tsconfig-paths/register -T "test/**/*.{ts,js}"`
```

Comment:
So I guess this is the key part that was missing in my PR?  Any idea why this works?  Do we need this `ts-node` step for other packages?  Just curious.

Reply:
Its initializing typescript parsing before loading ts files. https://github.com/TypeStrong/ts-node#mocha

>Do we need this `ts-node` step for other packages? Just curious.

I suspect not since its being added at the test runner level but I'm not 100%.